audb/core/api.py

```diff
@@ -595,11 +587,11 @@ def versions(
     """
     vs = []
     for repository in config.REPOSITORIES:
-        backend = audbackend.create(
+        backend = audbackend.access(
```

Comment:
Before we were calling `create()` as with the old implementation we did not care if the repository existed or not. It would then be created once the first file was uploaded. Now we have `create()` (which actually creates a repository) and `access()` which assumes a repository exists already. I now switched to `access()` as I think it's safer we assume a repository exists already.

Reply:
But this means we are now raising an error if the repository does not yet exists, correct?

If yes, we need to document that error in the docstring and should add an test for it.

Reply:
> But this means we are now raising an error if the repository does not yet exists, correct?

Also before we were raising an error, e.g. when the user had not rights to create the repository or the server did not exist. But it seems that we did not document those errors yet. E.g. here are the errors we currently document for `audb.load()`:

![image](https://github.com/audeering/audb/assets/10383417/63876597-58ba-4df0-a507-f4dd044775fa)

So maybe for now we only add a test and take care about the missing errors in another PR as this may involve a lot of changes.

Reply:
I agree and created https://github.com/audeering/audb/issues/326 to track this.

Reply:
I added the following test 'tests/test_publish.py::test_publish_error_repository_does_not_exist`